assets/lokomotive-kubernetes/bootkube/resources/bootstrap-manifests/bootstrap-apiserver.yaml

```diff
@@ -15,11 +15,12 @@ spec:
     - --advertise-address=$(POD_IP)
     - --allow-privileged=true
     - --anonymous-auth=false
-    - --authorization-mode=RBAC
+    - --authorization-mode=Node,RBAC
```

Comment:
In commit message, it should be `authorizer`, not `authoriser`, as in https://kubernetes.io/docs/reference/access-authn-authz/node/.